static/js/jira-redirect.js

```diff
@@ -0,0 +1,22 @@
+const params = new URLSearchParams(window.location.search.substring(1));
+const gheHost = params.get("serverUrl");
```

Comment:
Where are we passing this as a param? In `jira-server-url.js` it looks like we're setting `customData` to an empty object if no app exists `const customData = data.appExists ?  { serverUrl: gheServerURL } : {};` so from empty state we're never passing this value to the URL. And then in `github-app-creation.js` the only thing we're pulling from params is the jiraHost. So my guess here is that `gheHost` will be null.

We need to fix this and also add error handling in the event there isn't a gheHost.